test/manipulation_test.jl

```diff
@@ -0,0 +1,93 @@
+using Revise
+using JuMP,Gurobi,BilevelJuMP

```

Comment:
Can you test with an open source one?
Also, lets keep each test inside a function.
And use the testing functions instead of just prints.
Also, need to integrate to runtests file.